constraint/pkg/client/drivers/to_result.go

```diff
@@ -74,20 +73,10 @@ func ToResult(constraints map[ConstraintKey]*unstructured.Unstructured, r rego.R
 		Kind: keyMap["kind"],
 		Name: keyMap["name"],
 	}
-
 	constraint := constraints[key]
+
 	// DeepCopy the result so we don't leak internal state.
 	result.Constraint = constraint.DeepCopy()
 
-	enforcementAction, found, err := unstructured.NestedString(constraint.Object, "spec", "enforcementAction")
-	if err != nil {
-		return nil, err
-	}
-	if !found {
-		enforcementAction = apiconstraints.EnforcementActionDeny
```

Comment:
why do we need to remove this? we still need it for non-scope EA.

Reply:
`enforementAction` and `scopedEnforcementActions` are set [here ](https://github.com/open-policy-agent/frameworks/pull/403/files/2428b2195db8b85fd45f55d16fc1559c2ad00203#diff-2b54391e78f46b40ac71789b492e315a83cbd6b15716c000dcd4c5915d907802R741) - once driver is done with `query` and returns responses.

Reply:
where do you set if enforcementAction is not found, set to deny?

Reply:
It gets populated while adding constraint - [here](https://github.com/open-policy-agent/frameworks/pull/403/files/2428b2195db8b85fd45f55d16fc1559c2ad00203#diff-90ee2c69c3c3a9b661e6e71944b7cde7a50a126a49b694afedb30a8a4f50da81R102)